redbox-core/redbox/models/chain.py

```diff
@@ -117,6 +117,31 @@ class Source(BaseModel):
         default="",
     )
 
+    @field_validator("document_name", mode="before")
+    def validate_document_name(cls, value):
+        if value == "" or value is None:
```

Comment:
"not" is truthy/falsy so should be fine here, not super important but a bit more concise and inclusive
```suggestion
        if not value:
```